codalab/apps/web/views.py

```diff
@@ -747,14 +747,6 @@ def get_context_data(self, **kwargs):
             context['phase'] = phase
             context['groups'] = phase.scores()
 
-            bad_score_count, bad_scores = check_bad_scores(context['groups'])
```

Comment:
was it OK to remove this?

Reply:
Yeah we removed it from that view I believe?

Reply:
Yeah this is the correct place to remove the context from the other page we removed this feature from.

Reply:
Cool ty